strapi/README.md

```diff
@@ -6,7 +6,7 @@ This uses NPM, for compatibility reasons (See https://github.com/strapi/strapi/i
 
 To run it:
 
-```
+```text
```

Comment:
```suggestion
```sh
```

Reply:
@RaynalHugo The problem with using `sh` instead of `text` for declaring command-line stuff (not shell scripts) is that:

* it adds unwanted syntax coloring in some Markdown viewer

* it prevent use-cases where you want to display console output and/or prepend the line with a `$` or `#`. e.g.

```text
$ echo hello
hello
```